pkgs/by-name/ma/mangojuice/fix-vkbasalt-path.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/src/mangojuice.vala b/src/mangojuice.vala
+index 8c2cbef..46e7f73 100644
+--- a/src/mangojuice.vala
++++ b/src/mangojuice.vala
+@@ -2566,7 +2566,7 @@ public class MangoJuice : Adw.Application {
+     }
+ 
+     async bool check_vkbasalt_installed_async () {
+-        string[] paths = { "/usr/lib/libvkbasalt.so", "/usr/lib/x86_64-linux-gnu/libvkbasalt.so", "/usr/local/lib/libvkbasalt.so" };
++        string[] paths = { "/usr/lib/libvkbasalt.so", "/usr/lib/x86_64-linux-gnu/libvkbasalt.so", "/usr/local/lib/libvkbasalt.so", "@vkbasalt@" };
```

Comment:
Do we really should be trying all the other paths which are always wrong on NixOS? is there harm loading the nixpkgs vkbasalt on other systems?